twitter.js

```diff
@@ -216,9 +216,12 @@ Twitter.prototype.statuses = function(type, params, accessToken, accessTokenSecr
 			delete params.id;
 			method = "POST";
 			break;
-		case "update_with_media":
-			this.updateWithMedia(params, accessToken, accessTokenSecret, callback);
+		case "upload_media":
+			this.uploadMedia(params, accessToken, accessTokenSecret, callback);
 			return;
+		case "update_with_media":
+			callback("Upload with media type is deprecated. Use 'upload_media' instead");
```

Comment:
The word "deprecated" would be inaccurate given the code will not work. Suggest changing to "has been removed."